src/include/duckdb/storage/serialization/nodes.json

```diff
@@ -174,34 +174,6 @@
     ],
     "pointer_type": "none"
   },
-  {
-    "class": "SampleOptions",
```

Comment:
Instead of manually implementing the serialization - could we use the `"constructor"` option to pass the `seed` into the constructor and handle the logic for the `seed` switch there in the deserialization logic? Together with the `serialize_property` on the serialization side? e.g. you can serialize from a function call:

```
"serialize_property": "GetDateFormat()"
```